src/main/webapp/search-script.js

```diff
@@ -80,5 +80,47 @@ function addCourse() {
   url.searchParams.set("num-units", units);
   url.searchParams.set("term", termName);
   url.searchParams.set("school-name", schoolName);
-  fetch(url, { method: "POST" });
+  fetch(url, {method:"POST"});
+}
+
+function onSignIn(googleUser) {
+  const profile = googleUser.getBasicProfile();
+  document.getElementById("class-info").classList.remove("hidden");
+  document.getElementById("login-box").classList.add("hidden");
```

Comment:
You may want to wait to do this until after you know if the user was signed in or not